docs/bot/editor/actions/logic/google_calendar.md

```diff
@@ -0,0 +1,111 @@
+
+## 2. Authenticate to your Google account
+
+Click "Google connect" button and follow the instructions.
+
+:::tip
+Make sure to select all rights. Callbell bot will not work properly if any permissions are missing.
+
+<div class="text--center">
+    <img src={googleCredentials} width={500} />
+</div>
+
+You may also encounter this warning from Google:
+
+<div class="text--center">
+    <img src={googleCredentialsWarning} width={500} />
+</div>
+
+Which is just a temporary warning. In that case, make sure to continue by clicking "advanced" and then "Proceed with Callbell"
+:::
+
+## 3. Configuring the Action
+
+You will need to configure the integration by filling in the following fields:
+
+**Select calendar**: Choose the Google Calendar you want to interact with. Each Google account can have multiple calendars; the one labeled with an email address is your main calendar.
+
+**Select action**: Currently, `create event` is the only option available.
+
+**Event summary**: The title of the event as it will appear on the calendar.
+
+**Event month**: The month in which the event will be created.
+
+**Event day**: The day on which the event will be created.
+
+**Event starts at**: The time when the event will start. The required time format is: `HH:MM`.
+
+**Event duration (in minutes)**: The length of the event in minutes.
+
+**Send event updates to**: The recipients who will receive updates about the event. The options are: `None` or `All`.
+
+**Create meet link**: Option to generate a link for a virtual meeting associated with the event.
+
+:::tip
+You can use variables to dynamically populate these details
+:::
+
+## 4. Handling Success
+
+In case of a successful response from the API, the result will indicate the event creation status: `confirmed`. By default, this value will be saved as `Last webhook success`, but you can configure any variable you prefer for it. If `Create meet link` option is active, the URL of the meet will be stored in the variable.
+
+## 5. Handling Failure
+
+In case of an API failure, you can store the failure response in a dedicated variable for future analysis or retries.
+You can, for instance, imagine some logic that would check the failure and retry with a "jump" action.
+
+## 6. Example
+
+This is an example of how you can configure the action and use it with variables.
+
+1. In this example, we created 2 variables:
+
+<img src={googleCalendarVariables} width={500} />
+
+2. `Summary`, `month` and `duration` were set as fixed values.
+
+3. We used a combination of actions to dynamically retrieve the day:
+
+- [Send message](/bot/editor/actions/interaction/send_message) to send the message asking about which day is the best.
+- [Wait user's answer](/bot/editor/actions/interaction/wait_user_answer) to get the answer.
+- [Set variable](/bot/editor/actions/logic/set_variable) to save the result from the answer in our variable `day`.
```

Comment:
Maybe it would be cool to add an image showing how the combination will look like. WDYT?

Reply:
Good point